packages/form-system/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+
```

Comment:
might be accidental change?

Reply:
Definitely! Removed in the last commit. Thanks!